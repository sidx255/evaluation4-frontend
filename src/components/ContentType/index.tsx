import React from 'react';
import './ContentType.css';


const ContentType = () => {
  return (
    <div className='content-type'>
      <div className='content-type__header'>
        <div className='content-type__header__left'>
          <p className='content-type__header__left__text'>CONTENT TYPES</p>
        </div>
        <div className='content-type__header__right'>
          <p className='content-type__header__right__text'>CREATE</p>
        </div>
      </div>
      <div className='content-type__body'>
        <div className='content-type__body__left'>
          <p className='content-type__body__left__text'>CONTENT TYPES</p>
        </div>
        <div className='content-type__body__right'>
          <p className='content-type__body__right__text'>CREATE</p>
        </div>
      </div>
    </div>
  );
};

export default ContentType;