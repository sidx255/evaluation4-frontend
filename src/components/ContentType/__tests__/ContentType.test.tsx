
// snapshot test for ContentType component

import React from 'react';
import { render } from '@testing-library/react';

import ContentType from '../index';

describe('ContentType', () => {
  it('should render correctly', () => {
    const { container } = render(<ContentType />);
    expect(container).toMatchSnapshot();
  });
});
    