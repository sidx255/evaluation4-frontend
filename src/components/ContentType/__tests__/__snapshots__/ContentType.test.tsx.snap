// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentType should render correctly 1`] = `
<div>
  <div
    class="content-type"
  >
    <div
      class="content-type__header"
    >
      <div
        class="content-type__header__left"
      >
        <p
          class="content-type__header__left__text"
        >
          CONTENT TYPES
        </p>
      </div>
      <div
        class="content-type__header__right"
      >
        <p
          class="content-type__header__right__text"
        >
          CREATE
        </p>
      </div>
    </div>
    <div
      class="content-type__body"
    >
      <div
        class="content-type__body__left"
      >
        <p
          class="content-type__body__left__text"
        >
          CONTENT TYPES
        </p>
      </div>
      <div
        class="content-type__body__right"
      >
        <p
          class="content-type__body__right__text"
        >
          CREATE
        </p>
      </div>
    </div>
  </div>
</div>
`;
